docs/src/troubleshooting.md

```diff
@@ -569,3 +569,29 @@ VOLNAME=$(kubectl get pv -o json | \
 
 kubectl delete pod/$PODNAME pvc/$PODNAME pvc/$PODNAME-wal pv/$VOLNAME
 ```
+
+### Error while bootstrapping data directory
+
+If you encounter the issue that your database cannot be initialized because it crashes during the bootstraping process, it is likely that your hugepage settings are set wrong.
```

Comment:
Quote likely error message(s) that may indicate this fault, for search-ability and to help users.

Reply:
```
{"level":"info","ts":"2023-04-13T13:04:05Z","logger":"initdb","msg":"Bus error (core dumped)\nchild process exited with exit code 135\ninitdb: removing data directory \"/var/lib/postgresql/data/pgdata\"\n","pipe":"stderr","logging_pod":"cluster-example-initdb-1-initdb"}
```